services/telemetry/telemetry.go

```diff
@@ -65,6 +65,7 @@ const (
 	TRACK_PERMISSIONS_GENERAL       = "permissions_general"
 	TRACK_PERMISSIONS_SYSTEM_SCHEME = "permissions_system_scheme"
 	TRACK_PERMISSIONS_TEAM_SCHEMES  = "permissions_team_schemes"
+	TRACK_PERMISSIONS_SYSTEM_ROLES  = "permissions_system_roles"
```

Comment:
FYI @enelson720 this'll be a new table that'll be generated for server telemetry.

Reply:
@Adovenmuehle does it make sense to just make this part of the `permissions_system_scheme` table then? Maybe thats a question for @thefactremains

Reply:
Yeah that actually seems like it would simplify it. Good call.

Reply:
and maybe for completeness I can add the actual permissions for each of these roles to that table too

Reply:
The permissions for each role are hard-coded and common across all servers (of the same version), correct? In that case I don't think there's any huge need.

Reply:
If the admin modifies them then they would be different - I guess the question is _if_ permissions are modified do we care what the value of those permissions are? cc @thefactremains

Reply:
Disregard my last comment, yeah I think you should do that.

Reply:
> If the admin modifies them then they would be different (hence the `_permissions_modified` metrics) - I guess the question is _if_ permissions are modified do we care what the value of those permissions are? cc @thefactremains

Yes. We want to know that.